src/app/components/AuthWrapper/index.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ApiResponse } from 'apisauce';
+import { render, screen } from '@testing-library/react';
+
+import { LoginResponse } from '~screens/Login/types';
+import { ErrorResponse } from '~utils/types';
+
+import AuthWrapper from './index';
+
+describe('AuthWrapper Component', () => {
+  it('Renders successfully', async () => {
+    const mockOnSuccess = (data: ApiResponse<Partial<LoginResponse>, ErrorResponse>) => {};
```

Comment:
this should be a jest.fn and you can assert that is called with certain args, also it can be out of describe